Modules/ITS/src/ITSClusterTask.cxx

```diff
@@ -296,7 +304,8 @@ void ITSClusterTask::getJsonParameters()
   mDictPath = mCustomParameters["clusterDictionaryPath"];
   mRunNumberPath = mCustomParameters["runNumberPath"];
   mGeomPath = mCustomParameters["geomPath"];
-
+  mNThreads = stoi(mCustomParameters.find("nThreads")->second);
+  LOG(INFO) << "#################### mNThreads : " << mNThreads;
```

Comment:
it is certainly not a problem, but that would rather be a "debug" message than a "info" message.